modules/rubiconBidAdapter.js

```diff
@@ -716,6 +720,16 @@ export const spec = {
     }, []).sort((adA, adB) => {
       return (adB.cpm || 0.0) - (adA.cpm || 0.0);
     });
+
+    let fledgeAuctionConfigs = responseObj.component_auction_config?.map(config => {
+      return { config, bidId: config.bidId }
```

Comment:
Does this `bidId` returned by AE have to match the `bidId` prebid has for the bidRequest?

Where is DV+ getting this `bidId`?

Reply:
Yes. They're getting it from the fastlane param we pass in the adapter